SPECS/python-fields/python-fields.spec

```diff
@@ -0,0 +1,124 @@
+%global srcname fields
+
+Summary:        Container class boilerplate killer
+Name:           python-%{srcname}
+Version:        5.0.0
+Release:        9%{?dist}
+License:        BSD
+URL:            https://github.com/ionelmc/%{name}
```

Comment:
>%{name} [](http://example.com/codeflow?start=43&length=7)

Nit: we generally avoid macros in this specific tag, so we can keep the links clickable

Reply:
fixed:)